### ISSUE TITLE
Add Internationalization support

### ISSUE THREAD

**Issue by dpedoneze** [education/classroom#308]
I was looking at the code and found out that the texts were added as `String` and not using the _Internationalization (I18n) API_.

Would be good to have it implemented since it would be very useful afterwards.

**Comment by tarebyte** [comment]
@dpedoneze pull requests are welcome!

**Comment by johndbritton** [comment]
Yeah, I think this would be a great contribution. We've got a lot of other things to improve before we get to i18n, but if someone wants to add it I'd be happy to review the changes.

**Comment by beydogan** [comment]
@tarebyte have you started working on this? I can dive into this if you haven't.

**Comment by tarebyte** [comment]
@beydogan I have not, feel free to jump in! Can I assign you to this?

**Comment by beydogan** [comment]
@tarebyte yes sure

**Comment by fabriciojr** [comment]
@johndbritton @tarebyte Can I jump straight into this task?

**Comment by tarebyte** [comment]
> Can I jump straight into this task?

Feel free to @fabriciojr!

Just a few pieces of advice, that help us all better work together.

- Make a new issue with your game plan so we can discuss it (you can reference this issue).
- Open small atomic PRs instead of one large PR. This makes it easier for use to review your code and to make sure we can keep track of everything going on.

Looking forward to your contributions!
